control-plane/rest/openapi-specs/v0_api_spec.json

```diff
@@ -1 +1 @@
pe":"string"}],"tags":["Volumes"]}},"/volumes/{volume_id}/share/{protocol}":{"put":{"security":[{"JWT":[]}],"responses":{"200":{"description":"OK","schema":{"type":"string"}},"400":{"description":"Request Timeout","schema":{"$ref":"#/definitions/RestJsonError"}},"401":{"description":"Unauthorized","schema":{"$ref":"#/definitions/RestJsonError"}},"404":{"description":"Not Found","schema":{"$ref":"#/definitions/RestJsonError"}},"408":{"description":"Bad Request","schema":{"$ref":"#/definitions/RestJsonError"}},"412":{"description":"Precondition Failed","schema":{"$ref":"#/definitions/RestJsonError"}},"416":{"description":"Range Not satisfiable","schema":{"$ref":"#/definitions/RestJsonError"}},"422":{"description":"Unprocessable entity","schema":{"$ref":"#/definitions/RestJsonError"}},"500":{"description":"Internal Server Error","schema":{"$ref":"#/definitions/RestJsonError"}},"501":{"description":"Not Implemented","schema":{"$ref":"#/definitions/RestJsonError"}},"503":{"description":"Service Unavailable","schema":{"$ref":"#/definitions/RestJsonError"}},"504":{"description":"Gateway Timeout","schema":{"$ref":"#/definitions/RestJsonError"}},"507":{"description":"Insufficient Storage","schema":{"$ref":"#/definitions/RestJsonError"}}},"parameters":[{"description":"UUID of a mayastor volume","in":"path","name":"volume_id","required":true,"type":"string"},{"in":"path","name":"protocol","required":true,"type":"string","enum":["nvmf","iscsi"]}],"tags":["Volumes"]}},"/volumes{volume_id}/share":{"delete":{"security":[{"JWT":[]}],"responses":{"204":{"description":"OK"},"400":{"description":"Request Timeout","schema":{"$ref":"#/definitions/RestJsonError"}},"401":{"description":"Unauthorized","schema":{"$ref":"#/definitions/RestJsonError"}},"404":{"description":"Not Found","schema":{"$ref":"#/definitions/RestJsonError"}},"408":{"description":"Bad Request","schema":{"$ref":"#/definitions/RestJsonError"}},"412":{"description":"Precondition Failed","schema":{"$ref":"#/definitions/RestJsonError"}},"416":{"description":"Range Not satisfiable","schema":{"$ref":"#/definitions/RestJsonError"}},"422":{"description":"Unprocessable entity","schema":{"$ref":"#/definitions/RestJsonError"}},"500":{"description":"Internal Server Error","schema":{"$ref":"#/definitions/RestJsonError"}},"501":{"description":"Not Implemented","schema":{"$ref":"#/definitions/RestJsonError"}},"503":{"description":"Service Unavailable","schema":{"$ref":"#/definitions/RestJsonError"}},"504":{"description":"Gateway Timeout","schema":{"$ref":"#/definitions/RestJsonError"}},"507":{"description":"Insufficient Storage","schema":{"$ref":"#/definitions/RestJsonError"}}},"parameters":[{"description":"UUID of a mayastor volume","in":"path","name":"volume_id","required":true,"type":"string"}],"tags":["Volumes"]}}},"basePath":"/v0","securityDefinitions":{"JWT":{"name":"Authorization","type":"apiKey","in":"header","description":"Use format 'Bearer TOKEN'"}},"info":{"version":"v0","title":"Mayastor RESTful API"}}
```

Comment:
As an aside, I've been wondering, should we use a yaml format instead? more readable :)

Reply:
I think that is a good idea :+1: